src/pages/open.tsx

```diff
@@ -17,10 +17,6 @@ const Open: NextPage = () => {
     }
   }, [router, shouldUseNewCreation])
 
-  if (shouldUseNewCreation) {
-    return <></>
-  }
```

Comment:
This was causing hydration errors on my end but now there is a small loading flash which is also not ideal.

Reply:
Could we not render different links on the welcome page instead of replacing the route?

Reply:
Good idea! I will try that

Reply:
Seems to work good. I also had to update it for the safe app landing pages since they also link to the safe creation but I think its fine cc @iamacook